packages/shared_preferences/shared_preferences/README.md

```diff
@@ -17,9 +17,48 @@ Supported data types are `int`, `double`, `bool`, `String` and `List<String>`.
 
 ## Usage
 
-### Examples
+## SharedPreferences vs SharedPreferencesAsync vs SharedPreferencesWithCache
+
+As of the 2.3.0 update there are three available APIs that can be used in this package.
```

Comment:
Optional nit: I think this would be a bit more evergreen as "Starting with version 2.3.0 there are [...]"; referring to the "update" will seem a little weird when someone is reading this for the first time six months from now.